src/styles/components/_card.scss

```diff
@@ -0,0 +1,88 @@
+.card {
+  padding: 32px 16px 16px;
+  display: flex;
+  width: 200px;
+  box-sizing: border-box;
+  flex-direction: column;
+  align-items: center;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
+  color: $primeColor;
+  line-height: 14px;
+
+  @include fontParams(10px, 400);
+
+  &__image {
+    margin: 0 0 40px;
```

Comment:
If you need only one margin property it's better to use what you need instead of shorthand property.